src/app/util/attribute-storage.cpp

```diff
@@ -109,6 +109,10 @@ const uint16_t attributeManufacturerCodeCount                   = GENERATED_ATTR
 #define endpointNetworkIndex(x) fixedNetworks[x]
 #endif
 
+namespace {
+app::AttributeAccessInterceptor * attributeAccessInterceptors = nullptr;
```

Comment:
We really should avoid globals here like this. I know this is very much the theme for this file, but ideally, we should be hanging this off-of InteractionModelEngine, which is slightly better.

Reply:
We could hang it off of there too, sure.  This file needs to do the actual management, though, to handle removing things when endpoints are disabled, so it would be a lot more boilerplate to do that...

Reply:
I'm fine with it for now, since this needs a broader re-think of the globals in this file.